omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/ConceptFragmentController.java

```diff
@@ -0,0 +1,55 @@
+package org.openmrs.module.registrationapp.fragment.controller;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.openmrs.Concept;
+import org.openmrs.ConceptAnswer;
+import org.openmrs.api.ConceptService;
+import org.openmrs.module.registrationapp.RegistrationAppUtils;
+import org.openmrs.ui.framework.annotation.SpringBean;
+import org.springframework.web.bind.annotation.RequestParam;
+
+public class ConceptFragmentController {
+
+	public Collection<Map<String, ? extends Object>> getConcepts(
+			@SpringBean("conceptService") ConceptService conceptService,
+			@RequestParam(value = "term", required = true) String searchPhrase,
+			@RequestParam(value = "conceptId", required = true) String conceptId,
+			@RequestParam(value = "maxResults", required = false) String maxResults){
+		
+		Concept concept = RegistrationAppUtils.getConcept(conceptId, conceptService);
+		Collection<Map<String, ? extends Object>> collection = new ArrayList<Map<String,? extends Object>>();
+		Map<String, Object> conceptMap;
+		Locale locale = Locale.ENGLISH;
+		
+		Collection<ConceptAnswer> conceptAnswers = concept.getAnswers();
+		int answersSize = conceptAnswers.size();
+		int resultCount=0;
+		maxResults = (maxResults==null || maxResults.isEmpty()) ? "20" : maxResults;
```

Comment:
You can use apache common's StringUtils.isNotBlank to check for null, empty and whitespace characters